src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -209,9 +209,9 @@ functionsToRunOnDataTablesDrawEvent: [],
               "type": "POST"
           },
           dom:
-            "<'row hidden'<'col-sm-6 hidden-xs'i><'col-sm-6 hidden-print'f>>" +
+            "<'row hidden'<'col-sm-6'i><'col-sm-6 d-print-none'f>>" +
             "<'row'<'col-sm-12'tr>>" +
-            "<'row mt-2 '<'col-sm-6 col-md-4'l><'col-sm-2 col-md-4 text-center'B><'col-sm-6 col-md-4 hidden-print'p>>",
+            "<'row mt-2 d-print-none '<'col-sm-12 col-md-4'l><'col-sm-0 col-md-4 text-center'B><'col-sm-12 col-md-4 'p>>",
```

Comment:
Includes `print` fixes - looks like we didn't change the classes when moving from Bootstrap 3 to Bootstrap 4.

Also makes the bulk buttons show up nicely on XS. Centered.